src/ParaTest/Runners/PHPUnit/TestMethod.php

```diff
@@ -66,6 +78,6 @@ protected function prepareOptions($options)
      */
     public function getTestMethodCount()
     {
-        return 1;
+        return count($this->filters);
```

Comment:
Also I'm thinking about renaming of this class because actually it's a set of tests for test file controlled by filter and filter can match with different methods and different datasets.